Sources/Shared/Models/PublishJobConnection.swift

```diff
@@ -25,15 +25,18 @@
 
 @objcMembers
 public class PublishJobConnection: VIMConnection {
-    public var publishBlockers: PublishJobBlockers?
-    public var publishConstraints: PublishJobConstraints?
+    public private(set) var publishBlockers: PublishJobBlockers?
+    public private(set) var publishConstraints: PublishJobConstraints?
+    public private(set) var publishDestinations: PublishDestinations?
```

Comment:
`PublishDestinations` class with multiple `PublishJobDestination` for each destination. But by blueprint and in response just `bool` values. 

https://github.vimeows.com/Vimeo-API/design/pull/148/files#diff-4dafa2f9b34cb46c64cb8ef8d9694149R157
```
            - `publish_destinations` (object) - Information on which social networks this video was published to.
	                - `facebook` `true` (boolean, required) - Whether the video was ever published to Facebook.
	                - `youtube` `true` (boolean, required) - Whether the video was ever published to YouTube.
	                - `linkedin` `false` (boolean, required) - Whether the video was ever published to LinkedIn.
	                - `twitter` `false` (boolean, required) - Whether the video was ever published to Twitter.
```

Seems wrong 😕